nipype/interfaces/ants/registration.py

```diff
@@ -440,13 +440,15 @@ class Registration(ANTSCommand):
 
     >>> # Test multiple metrics per stage
     >>> reg5 = copy.deepcopy(reg)
+    >>> reg5.inputs.fixed_image = [ 'fixed1.nii', 'fixed2.nii' ]
+    >>> reg5.inputs.moving_image = [ 'moving1.nii', 'moving2.nii' ]
     >>> reg5.inputs.metric = ['CC', ['CC', 'Mattes']]
     >>> reg5.inputs.metric_weight = [1, [.5]*2]
     >>> reg5.inputs.radius_or_number_of_bins = [4, [32]*2]
     >>> reg5.inputs.sampling_strategy = ['Random', None] # use default strategy in second stage
     >>> reg5.inputs.sampling_percentage = [0.05, [0.05, 0.10]]
     >>> reg5.cmdline
-    'antsRegistration --collapse-output-transforms 0 --dimensionality 3 --initial-moving-transform [ trans.mat, 1 ] --initialize-transforms-per-stage 0 --interpolation Linear --output [ output_, output_warped_image.nii.gz ] --restore-state trans.mat --save-state trans.mat --transform Affine[ 2.0 ] --metric CC[ fixed1.nii, moving1.nii, 1, 4, Random, 0.05 ] --convergence [ 1500x200, 1e-08, 20 ] --smoothing-sigmas 1.0x0.0vox --shrink-factors 2x1 --use-estimate-learning-rate-once 1 --use-histogram-matching 1 --transform SyN[ 0.25, 3.0, 0.0 ] --metric CC[ fixed1.nii, moving1.nii, 0.5, 32, None, 0.05 ] --metric Mattes[ fixed1.nii, moving1.nii, 0.5, 32, None, 0.1 ] --convergence [ 100x50x30, 1e-09, 20 ] --smoothing-sigmas 2.0x1.0x0.0vox --shrink-factors 3x2x1 --use-estimate-learning-rate-once 1 --use-histogram-matching 1 --winsorize-image-intensities [ 0.0, 1.0 ]  --write-composite-transform 1'
+    'antsRegistration --collapse-output-transforms 0 --dimensionality 3 --initial-moving-transform [ trans.mat, 1 ] --initialize-transforms-per-stage 0 --interpolation Linear --output [ output_, output_warped_image.nii.gz ] --restore-state trans.mat --save-state trans.mat --transform Affine[ 2.0 ] --metric CC[ fixed1.nii, moving1.nii, 1, 4, Random, 0.05 ] --convergence [ 1500x200, 1e-08, 20 ] --smoothing-sigmas 1.0x0.0vox --shrink-factors 2x1 --use-estimate-learning-rate-once 1 --use-histogram-matching 1 --transform SyN[ 0.25, 3.0, 0.0 ] --metric CC[ fixed1.nii, moving1.nii, 0.5, 32, None, 0.05 ] --metric Mattes[ fixed2.nii, moving2.nii, 0.5, 32, None, 0.1 ] --convergence [ 100x50x30, 1e-09, 20 ] --smoothing-sigmas 2.0x1.0x0.0vox --shrink-factors 3x2x1 --use-estimate-learning-rate-once 1 --use-histogram-matching 1 --winsorize-image-intensities [ 0.0, 1.0 ]  --write-composite-transform 1'
```

Comment:
Here is the command line for ants, and it looks correct to me:
(with metric= ['Mattes', ['Mattes','CC'] ] )

antsRegistration \
 \
  --collapse-output-transforms 0 \
  --dimensionality 3 \
  --initial-moving-transform [ trans.mat, 1 ] \
  --initialize-transforms-per-stage 0 \
  --interpolation Linear \
  --output [ output_, output_warped_image.nii.gz ] \
  --restore-state trans.mat \
  --save-state trans.mat \
 \  ##
 \  ## STAGE 1 Affine transform registration, using Mattes, at low shrink factors
  --transform Affine[ 2.0 ] \
  --metric Mattes[ fixed1.nii, moving1.nii, 1, 4, Random, 0.05 ] \
  --convergence [ 1500x200, 1e-08, 20 ] \
  --smoothing-sigmas 1.0x0.0vox \
  --shrink-factors 2x1 \
  --use-estimate-learning-rate-once 1 \
  --use-histogram-matching 1 \
 \  ##
 \  ## STAGE 2 SyN based transform estimation, using Mattes for fixed1/moving1 and
 \  ##                                         using     CC for fixed2/moving2
  --transform SyN[ 0.25, 3.0, 0.0 ] \
  --metric Mattes[ fixed1.nii, moving1.nii, 0.5, 32, None, 0.05 ] \
  --metric CC[ fixed2.nii, moving2.nii, 0.5, 32, None, 0.1 ] \
  --convergence [ 100x50x30, 1e-09, 20 ] \
  --smoothing-sigmas 2.0x1.0x0.0vox \
  --shrink-factors 3x2x1 \
  --use-estimate-learning-rate-once 1 \
  --use-histogram-matching 1 \
 \
  --winsorize-image-intensities [ 0.0, 1.0 ]  \
  --write-composite-transform 1